packages/shared/src/components/fields/ContextMenu.tsx

```diff
@@ -48,12 +48,13 @@ export interface MenuItemProps<
 
 interface ContextMenuProps extends Omit<MenuProps, 'children'> {
   options: MenuItemProps[] | ContextMenuDrawerItem[];
-  isOpen: boolean;
+  isOpen?: boolean;
 }
 
 export default function ContextMenu({
   options,
   onHidden,
+  isOpen,
```

Comment:
This makes it so you miss `isOpen` in the `PortalMenu` is that ok?

Reply:
and why chosen to make it optional now?

Reply:
Yes. React was screaming that it is not needed to the other element.